libkbfs/journal_server_test.go

```diff
@@ -747,6 +748,45 @@ func TestJournalServerEnableAuto(t *testing.T) {
 	require.Len(t, tlfIDs, 1)
 }
 
+func TestJournalServerPublicTLF(t *testing.T) {
+	tempdir, ctx, cancel, config, _, jServer := setupJournalServerTest(t)
+	defer teardownJournalServerTest(t, tempdir, ctx, cancel, config)
+
+	err := jServer.EnableAuto(ctx)
+	require.NoError(t, err)
+
+	status, tlfIDs := jServer.Status(ctx)
+	require.True(t, status.EnableAuto)
+	require.Zero(t, status.JournalCount)
+	require.Len(t, tlfIDs, 0)
+
+	h, err := ParseTlfHandle(
+		ctx, config.KBPKI(), nil, "test_user2", tlf.Public)
+	require.NoError(t, err)
+
+	// Access someone else's public TLF, which should NOT create a
+	// journal automatically.
+	_, err = config.MDOps().GetIDForHandle(ctx, h)
+	require.NoError(t, err)
+
+	status, tlfIDs = jServer.Status(ctx)
+	require.True(t, status.EnableAuto)
```

Comment:
What is `EnableAuto`?

Reply:
`EnableAuto` means that journals are created automatically when you access them (now the default for everyone, but it didn't used to be).

Reply:
All right, so this check is essentially to make sure that the `EnableAuto=false` switch isn't the cause `status.JournalCount==0`.

Reply:
Right.